packages/dynamic_forms/lib/src/parser/parser_node.dart

```diff
@@ -90,8 +98,10 @@ abstract class ParserNode {
   static String emptyString() => "";
   static String convertToString(String x) => x;
   static bool convertToBool(String x) => x?.toLowerCase() == "true";
+  static int convertToColor(String x) => int.parse(x);
```

Comment:
You work with the colors as with the int, so I would probably remove this line and in the parser, I would use getIntValue method.